acvm-repo/acvm/src/compiler/transformers/mod.rs

```diff
@@ -522,3 +552,103 @@ where
+ASSERT w32 = 60
+ASSERT w33 = 33
+ASSERT w34 = 31
+ASSERT w0 = 16777216*w35 + 4294967296*w36 + 1099511627776*w37 + 281474976710656*w38 + 72057594037927936*w39 + 18446744073709551616*w40 + 4722366482869645213696*w41 + 1208925819614629174706176*w42 + 309485009821345068724781056*w43 + 79228162514264337593543950336*w44 + 20282409603651670423947251286016*w45 + 5192296858534827628530496329220096*w46 + 1329227995784915872903807060280344576*w47 + 340282366920938463463374607431768211456*w48 + 87112285931760246646623899502532662132736*w49 + 22300745198530623141535718272648361505980416*w50 + 5708990770823839524233143877797980545530986496*w51 + 1461501637330902918203684832716283019655932542976*w52 + 374144419156711147060143317175368453031918731001856*w53 + 95780971304118053647396689196894323976171195136475136*w54 + 24519928653854221733733552434404946937899825954937634816*w55 + 6277101735386680763835789423207666416102355444464034512896*w56 + 1606938044258990275541962092341162602522202993782792835301376*w57 + 411376139330301510538742295639337626245683966408394965837152256*w58 + 105312291668557186697918027683670432318895095400549111254310977536*w59 + 26959946667150639794667015087019630673637144422540572481103610249216*w60 + 6901746346790563787434755862277025452451108972170386555162524223799296*w61 + 1766847064778384329583297500742918515827483896875618958121606201292619776*w62 + 2040124
+ASSERT w62 = w1
+ASSERT w61 = w2
+ASSERT w60 = w3
+ASSERT w59 = w4
+ASSERT w58 = w5
+ASSERT w57 = w6
+ASSERT w56 = w7
+ASSERT w55 = w8
+ASSERT w54 = w9
+ASSERT w53 = w10
+ASSERT w52 = w11
+ASSERT w51 = w12
+ASSERT w50 = w13
+ASSERT w49 = w14
+ASSERT w48 = w15
+ASSERT w47 = w16
+ASSERT w46 = w17
+ASSERT w45 = w18
+ASSERT w44 = w19
+ASSERT w43 = w20
+ASSERT w42 = w21
+ASSERT w41 = w22
+ASSERT w40 = w23
+ASSERT w39 = w24
+ASSERT w38 = w25
+ASSERT w37 = w26
+ASSERT w36 = w27
+ASSERT w35 = w28
+ASSERT w29 = 31
+ASSERT w30 = 33
+ASSERT w31 = 60
+"#;
+
+        let acir = Circuit::from_str(formatted_acir).unwrap();
+        assert!(CircuitSimulator::default().check_circuit(&acir).is_none());
+        let expression_width = ExpressionWidth::Bounded { width: 4 };
+        let acir_opcode_positions = vec![
+            0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
+            24, 25, 26, 27, 28, 29, 29, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43,
+            44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64,
+        ];
+        let mut brillig_side_effects = BTreeMap::new();
+        brillig_side_effects.insert(BrilligFunctionId(0), false);
+
+        let (_, _, opcodes_hash_stabilized) = transform_internal(
+            acir,
+            expression_width,
+            acir_opcode_positions.clone(),
+            &brillig_side_effects,
+            Some(3),
```

Comment:
```suggestion
            None,
```
Are we just trying to test that our max transformer passes is not enough? Should we just use `None` or that constant here?